### DISCUSSION TITLE
Есть ли возможность или будет ли возможности импортированные моделей персонажей правильных костей для движков Unity и так далее?

### DISCUSSION THREAD

**Discussion by DanilSemenov41** [PavelBlend/blender-xray#628] [Q&A]
Здравствуйте, я вот хочу узнать возможно ли сделать исправленные кости? Чтобы возможно было в них ходить в VR (Виртуальная Реальность) А то есть одна проблема с костями, из-за того что имеет кости из движка X-Ray.
![Снимок экрана (1994)](https://github.com/PavelBlend/blender-xray/assets/133515611/2d552f83-5013-46ea-a3a0-acd3b3be6d0c)

![Снимок экрана (2103)](https://github.com/PavelBlend/blender-xray/assets/133515611/29db0253-039c-47df-854c-f11d3134eb02)

**Comment by PavelBlend** [comment]
@DanilSemenov41 Привет.

А каким критериям должны соответствовать кости в unity? Должны быть выстроены в цепочку? Если да, то есть возможность создать такой скелет. Здесь есть ссылка на видео в ютубе:
https://github.com/PavelBlend/blender-xray/releases/tag/v2.8.0

Ещё есть возможность создать скелет для деформации. Он имеет суффикс `weight`. Вот описание:
https://github.com/PavelBlend/blender-xray/discussions/602

Нужно использовать оператор `Create Connected Bones`. После его запуска, нужно в параметре `Source Armature` этого оператора указать арматуру сталкера. После этого нажать ок и будут созданы два скелета. Тебе будет нужен с суффиксом `weight`. Его нужно указать в модификаторе арматуры у меша и сделать меш потомком этого скелета.

Этот оператор в 3д окне в N панеле, у тебя на скриншоте он виден.

**Reply by DanilSemenov41** [reply]
Вроде всё так же сделал, но вот когда экспортировал для Unity руки как и были так и остались деформированные. 
Не так и просто переместить модель персонажа в Unity.

**Reply by PavelBlend** [reply]
@DanilSemenov41 я в юнити не разбираюсь. Я даже не знаю, какие у него требования к скелету. Лучше спроси на форумах или в сообществах по юнити. Я его ни разу не запускал даже.

**Reply by DanilSemenov41** [reply]
Ладно, благодарю за помощь всё таки.
